spec/models/scheduled_message_spec.rb

```diff
@@ -23,4 +23,14 @@
       expect(ScheduledMessage.date_time_ordering).to match_array([m1, m2, m3, m4, m5])
     end
   end
+
+  describe 'active messages' do
```

Comment:
hey! thanks for adding tests for this. you mind updating this `describe` block to reflect the name of the method? eg:

`describe '.active' do`